src/zepto.js

```diff
@@ -849,7 +849,16 @@ var Zepto = (function() {
       // arguments can be nodes, arrays of nodes, Zepto objects and HTML strings
       var argType, nodes = $.map(arguments, function(arg) {
             argType = type(arg)
-            return argType == "object" || argType == "array" || arg == null ?
+            if (argType == "array"){
+              var arr = []
```

Comment:
I think the `var arr` should be on the first line of the function, because it would get hoisted there anyway. (I'd leave the `arr = []` assignment in the block, tho.